docs: Publish hotfix of multi dimension support on StatsD sink note (#2372)
tomkerkhove/promitor: changelog/content/experimental/unreleased.md

#### Scraper

> - {{% tag fixed %}} Fix support for multi dimension scraping on statsD sink.

Reviewer: ```suggestion
- {{% tag fixed %}} Provide missing functionality for multi-dimension scraping in StatsD sink.
```
